main/webapp/modules/core/scripts/reconciliation/recon-dialog.js

```diff
@@ -108,29 +95,49 @@ ReconDialog.prototype._cleanDialog = function() {
 
 ReconDialog.prototype._populateDialog = function() {
   var self = this;
+  self._elmts.serviceList.empty();
 
   var services = ReconciliationManager.getAllServices();
   if (services.length > 0) {
+    
     var renderService = function(service) {
       var record = {
           service: service,
           handler: null
       };
 
-      record.selector = $('<a>')
-      .attr("href", "javascript:{}")
-      .addClass("recon-dialog-service-selector")
-      .text(service.name)
-      .appendTo(self._elmts.serviceList)
-      .on('click',function() {
-    	self._toggleServices();
-        self._selectService(record);
-      });
+      var label = $('<label>')
+        .addClass('recon-dialog-service-list-element')
+        .appendTo(self._elmts.serviceList);
+
+      record.selector = $('<input type="radio" name="service-choice">')
+        .addClass("recon-dialog-service-selector")
+        .val(service.name) 
+        .appendTo(label)
+
+      var mainSpan=$('<span>')
+        .addClass("recon-service-entry")
+        .appendTo(label);
+
+      var divElement=$('<div>')
+        .text(service.name)
+        .appendTo(mainSpan);
+
+      var divElementURL = $('<div>')
+        .css('color','lightgrey')
```

Comment:
In the Wikibase dialog, we use `color: grey`, so I would use that here too (for consistency and to have a slightly better contrast with the white background). Also, it would be worth storing this in a CSS class (instead of inline CSS). Ideally this color should be based on a CSS variable, which you could also apply in the Wikibase selection dialog in the same go.

Reply:
Hi Antonin, 
I am confused about where should this be applied in the Wikibase selection dialog?

Reply:
It's in `wikibase-dialog.css`, here:
https://github.com/OpenRefine/OpenRefine/blob/ddd0137c40b6ebfd1f442f0c2a68b02f3632ba40/extensions/wikibase/module/styles/dialogs/wikibase-dialog.css#L60-L63